db/migrate/20191030214642_create_digital_services_and_forms.rb

```diff
@@ -0,0 +1,23 @@
+class CreateDigitalServicesAndForms < ActiveRecord::Migration[5.0]
```

Comment:
migraciu by som prefixoval modulom, kvoli koliziam v nazvoch (podobne ako pri ITMS)

```suggestion
class CreateUpvsDigitalServicesAndForms < ActiveRecord::Migration[5.0]
```